gateway/src/main/java/io/zeebe/Gateway.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.zeebe;
+
+import io.grpc.Server;
+import io.grpc.ServerBuilder;
+import io.zeebe.broker.client.ZeebeClient;
+import io.zeebe.util.ZbLogger;
+import java.io.IOException;
+import java.time.Duration;
+import org.slf4j.Logger;
+
+public class Gateway {
+
+  private static final Logger LOG = new ZbLogger("io.zeebe.gateway");
+  private static final int CLUSTER_CLIENT_API_PORT = 26501;
```

Comment:
Let's create an issue to add more configuration options for the gateway and that the standalone one can be configured, i.e. the properties from the broker-client probably have to be renamed, maybe we want a toml file configuration again etc.

Reply:
#1135